packages/webdriver-utils/src/providers/genericProvider.js

```diff
@@ -278,4 +286,16 @@ export default class GenericProvider {
     }
     return ignoredElementsArray;
   }
+
+  async getHeaderFooter() {
+    const devicesConfig = (await request(DEVICES_CONFIG_URL)).body;
```

Comment:
lets call this once and keep it present in the local SDK server, this way per session this file would be cached.